nltk/text.py

```diff
@@ -530,16 +534,50 @@ def dispersion_plot(self, words):
         from nltk.draw import dispersion_plot
 
         dispersion_plot(self, words)
-
-    def generate(self, words):
-        """
-        Issues a reminder to users following the book online
-        """
-        import warnings
-
-        warnings.warn(
-            'The generate() method is no longer available.', DeprecationWarning
-        )
+        
+    def build_ngram_lm(self, tokenized_sents, n=3):
```

Comment:
Do you expect this to be part of the public API of `Text`? If not, I would say it makes sense to make it a private helper function in this module called `_train_default_ngram_lm`.

Reply:
👍

Reply:
Changed to `self. _train_default_ngram_lm()`.